pom.xml

```diff
@@ -108,14 +108,14 @@
         <module>utils</module>
         <module>conformance</module>
         <module>standardization</module>
-        <module>dataModel</module>
+        <module>data-model</module>
         <module>examples</module>
         <module>dao</module>
         <module>menas</module>
         <module>migrations</module>
         <module>migrations-cli</module>
         <module>external/kafka</module>
-        <module>external/fixedWidth</module>
+        <module>external/fixed-width</module>
```

Comment:
Do you remember why these modules are 'external'?

Reply:
Not really, I think because they were not originally meant to be part of the project, but got smuggled out during the process of open sourcing it. Maybe we can factor them out as well? Have to double check where they might be used and if there is any coupling. They might end up being top level modules as well.

Reply:
This investigation might worth a separate task on GutHub. I can log it.